libsolidity/codegen/ir/IRGeneratorForStatements.cpp

```diff
@@ -2520,16 +2520,25 @@ void IRGeneratorForStatements::appendExternalFunctionCall(
 		</noTryCall>
 		<?+retVars> let <retVars> </+retVars>
 		if <success> {
+			let <returnDataSize> := returndatasize()
```

Comment:
`returndatasize()` is an opcode that is not always available, it can only be used in never evm versions. You need to check `supportsreturnData`

Reply:
It's probably better to start with `<estimatedReturnSize>` instead.